### ISSUE TITLE
Proposition : support DateTime type 

### ISSUE THREAD

**Issue by Sraime** [graphaware/neo4j-php-ogm#117]
Hello !
I actually work on a project where users have to set their birthday on registration. 
As you know, in Symfony, validators are used to check if setting values are correct. The best way for the dating values is the DateTime type which is compatible with GreaterThan and LessThan asserts. 
```
/**
* @var string
* @Assert\LessThan("-10 years", message="user.dateofbirth.tooyoung")
* @Assert\GreaterThan("-80 years", message="user.dateofbirth.tooold")
*/
 private $dateOfBirth;
```
This type can’t be use with the OGM because DateTime cannot by convert to a primitive type in Neo4j.
Dates are very use in many application and it’s could be nice to offer the possibility to use DateTime (and then Assert validators) instead of a simple timestamp.

I work on it and I will propose an implementation very soon in a pull request.

**Comment by ikwattro** [comment]
Yes it is planned as part of next RC's

**Comment by ikwattro** [comment]
Fixed by #128
